i18n/en.json

```diff
@@ -36,7 +36,7 @@
     "softkey-search": "Search",
     "search-language-placeholder": "Search languages",
     "softkey-menu": "Menu",
-    "search-placeholder": "Search",
+    "search-placeholder": "Search Wikipedia",
```

Comment:
this key also being used in language page, do we want to apply this to all?

Reply:
There is `search-language-placeholder` above, shouldn't this one be used on the language page?

Reply:
yes, language page should use `search-language-placeholder`